assets/scss/templates/_guidelines.scss

```diff
@@ -588,22 +592,741 @@ body {
   }
 }
 
-/* Content Wrapper */
+/* Content Wrapper Inicio */
 
 .dpsg-content-wrap {
   flex: 1;
-  padding: $dp-spaces--lv7 $dp-spaces--lv5;
+  padding: $dp-spaces--lv0;
   opacity: 0;
   animation: fadeInLeft 0.5s linear forwards;
   animation-delay: 4.5s;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  position: relative;
+
+  h3 {
+    font-size: $dp-sizing--lvl7;
+    font-weight: 300;
+    display: inline-block;
+    color: #000;
+    position: relative;
+    font-family: 'Raleway';
+    line-height: 1.2em;
+  }
+
+  p {
+    font-size: 1em;
+    max-width: 80%;
+    margin: 0 0 16px;
+    line-height: 1.5em;
+  }
+
+  a {
+    text-decoration: underline;
+    transition: all 0.1s ease-in-out;
+  }
+
+  /* introduction hero */
+
+  .dpsg-content-intro {
+    padding: 0;
+    width: 100%;
+    position: relative;
+    background: #f1ede1 url(../img/pattern_01.png);
```

Comment:
Please check the color here, if it is only used here perhaps you could have a set of colors for this screen only.